src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/BotSegments/Models/BotActionType.cs

```diff
@@ -11,6 +11,9 @@ public enum BotActionType
         Mouse_CVImage,
         Mouse_CVText,
         Mouse_ObjectDetection,
+        RestartGame,
+        QuitGame,
+        ResetForNextTest
```

Comment:
I would rather name this `ResetGame`, but that is too close to `RestartGame` to be completely clear.

Ultimately, I don't like `Reset` existing separately from `Restart`, but that was the only way to fufill @vontell 's ask that the custom impl be testable in the editor if desired.